drivers/hud_lsa/app/models/hud_lsa/generators/fy2023/lsa.rb

```diff
@@ -45,6 +45,8 @@ def lookback_stop_date
 
     def run!
       setup_notifier('LSA')
+      return unless preflight_passes?
```

Comment:
This is fine, suggest setting `@failed` within the `preflight_passes?` method so it isn't a side-effect
```
@failed = !preflight_passes?
```

Reply:
Thanks!